analyzers/src/SonarAnalyzer.Common/Helpers/GeneratedCodeRecognizer.cs

```diff
@@ -92,11 +86,7 @@ private bool HasAutoGeneratedComment(SyntaxNode root)
 
             return firstToken.LeadingTrivia
                 .Where(IsTriviaComment)
-                .Any(trivia =>
-                {
-                    var commentText = trivia.ToString().ToUpperInvariant();
-                    return Array.Exists(AutoGeneratedCommentParts, commentText.Contains);
-                });
+                .Any(trivia => AutoGeneratedCommentParts.Exists(part => trivia.ToString().IndexOf(part, StringComparison.OrdinalIgnoreCase) >= 0));
```

Comment:
Use [Contains](https://learn.microsoft.com/en-us/dotnet/api/system.string.contains?view=net-8.0#system-string-contains(system-string-system-stringcomparison)) instead of `IndexOf`.

Reply:
That is not possible. That overload is `netstandard2.1`. In the Remarks section, `IndexOf` is the recommended alternative.